swap_meet/vendor.py

```diff
@@ -1,2 +1,51 @@
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if inventory == None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        for i in range(len(self.inventory)):
+            if self.inventory[i] == item:
+                return self.inventory.pop(i)
+        return False
+
+    def get_by_category(self, category):
+        return [item for item  in self.inventory if item.category == category]
+    
+    def swap_items(self, Vendor, my_item, their_item):
```

Comment:
by passing in `Vendor` as a parameter, we have overwritten the `Vendor` class in the scope of this function. It doesn't cause a problem here, but in general we should avoid passing in class names as parameters. Instead, choose something like `other`, `friend`, or even `vendor` (lowercase)